packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -19,11 +19,14 @@ export interface InterventionPolicyGroup {
 	name: string;
 	parameter: string;
 	startTime: number;
+	endTime: number;
 	lowerBound: number;
 	upperBound: number;
 	initialGuess: number;
 	isActive: boolean;
 	paramValue: number;
```

Comment:
Do we need to store this here?
Param value should now be either
intervention.dynamicIntervention.value
or
intervention.staticIntervention.value

Reply:
This will be done by me in future PR